src/qt/rpcconsole.cpp

```diff
@@ -357,10 +356,10 @@ void RPCConsole::setClientModel(ClientModel *model)
 
         // create peer table context menu actions
         QAction* disconnectAction = new QAction(tr("&Disconnect Node"), this);
-        QAction* banAction1h      = new QAction(tr("Ban Node for") + " " + tr("1 &hour"), this);
-        QAction* banAction24h     = new QAction(tr("Ban Node for") + " " + tr("1 &day"), this);
-        QAction* banAction7d      = new QAction(tr("Ban Node for") + " " + tr("1 &week"), this);
-        QAction* banAction365d    = new QAction(tr("Ban Node for") + " " + tr("1 &year"), this);
+        QAction* banAction1h      = new QAction(tr("Ban Node for") + " " + tr("1 hour"), this);
```

Comment:
If I remember correctly we had that discussion during the creation of the pull a few weeks ago, but you didn't want to add these changes :). AFAIK this isn't working in context menus.